proxies/live/apiproxy/resources/py/check-app-enabled-endpoint.py

```diff
@@ -1,12 +1,13 @@
 path_suffix = flow.getVariable("proxy.pathsuffix").lower()
 request_verb = flow.getVariable("request.verb").lower()
 
-requested_endpoint = (path_suffix, request_verb)
-
-
-auth_forbidden = requested_endpoint in [
+blocked_resources = [
     ("/fhir/r4/relatedperson", "get"),
     ("/fhir/r4/questionnaireresponse", "post"),
 ]
 
```

Comment:
Could this break? as auth_forbidden may not be set. It's never set to False as a default so could only be True or undefined 
Same for the other script